SU2_CFD/src/output/COutput.cpp

```diff
@@ -853,29 +711,27 @@ void COutput::WriteToFile(CConfig *config, CGeometry *geometry, OUTPUT_TYPE form
       break;
   }
 
-  if (fileWriter != nullptr){
+  if (fileWriter != nullptr) {
 
     /*--- Write data to file ---*/
 
     fileWriter->WriteData(fileName);
 
     su2double BandWidth = fileWriter->GetBandwidth();
 
-    /*--- Write data with iteration number to file ---*/
+    /*--- Write data with iteration number to file if required ---*/
 
-    if (!filename_iter.empty() && !config->GetWrt_Restart_Overwrite()){
+    if (!filename_iter.empty()) {
```

Comment:
It was enough to base it just on the filename :) but in the process I simplified a bit more